app/controllers/results_controller.rb

```diff
@@ -56,7 +60,14 @@ def edit
     @extra_marks_percentage = @result.extra_marks.percentage
     @marks_map = Hash.new
     @old_marks_map = Hash.new
-    @mark_criteria = @assignment.get_criteria(:ta)
+
+    if !@assignment.pr_assignment.nil?
```

Comment:
I believe there's a helper in `Assignment` which checks if an assignment has a peer review